dask_gdf/accessor.py

```diff
@@ -0,0 +1,86 @@
+from toolz import partial 
+import pygdf as gd
+from pygdf.series import DatetimeProperties
+
+
+class Accessor(object):
+    """
+    Base class for Accessor objects dt, str, cat. 
+    Notes
+    -----
+    Subclasses should define the following attributes:
+    * _accessor
+    * _accessor_name
+    """
+    _not_implemented = set()
+
+    def __init__(self, series):
+        from .core import Series
+        if not isinstance(series, Series):
+            raise ValueError('Accessor cannot be initialized')
+        self._series = series
+        self._validate(series)
+
+    def _validate(self, series):
+        pass
```

Comment:
Should this raise a `NotImplementedError`? Or is it OK if a derived class does not actually implement validation?